docs/accessControl/index.md

```diff
@@ -10,6 +10,23 @@ In PrairieLearn there are two levels of access control for students to gain acce
 
 2. Second, a student must also have access to the specific **assessment**. This access is granted with the `allowAccess` rules in [infoAssessment.json](../assessment/index.md). Even if a student has been granted access to an assessment, however, they will only be able to actually access it if they also have access to the course instance (see previous item).
 
+## Publishing configuration for course instances
```

Comment:
Very soon I think we'll want totally separate docs for course instances and assessments, putting this all in one file is very confusing.

Reply:
I agree that we should split out those docs eventually.